### ISSUE TITLE
Keyboard Accessibility Issue in the gridx

### ISSUE THREAD

**Issue by rajendrakesarwani** [oria/gridx#279]
Steps to Reproduce.
1-> I used "formatter" in the column structure and returning a link.
2-> Since I don't have mouse so I pressed tab to reach the cell where link is available.
3-> After reaching to the cell I am unable to open the link.
4-> Can you please let me know the way I could open the link available in the column's cell.

Thanks & Regards
Rajendra Kumar

**Comment by chenzhh** [comment]
This issue is not valid defect.
You can reach the cell by pressing arrow keys, then press F2, the link inside the cell will get focused. You can open the link on ENTER key.

**Comment by smathirajan** [comment]
Hi,
Please find the issues below.
#1 tabbing is not working for the cells but arrow keys are working to navigate to the cells.
#2 I have a formatted column and I have a icon in that cell. tabbing is not working for that icon in that cell though i tried with F2 key.
Hence the keyboard accessibility is failed.

**Comment by Sreelalitha2** [comment]
in my scenario, could see issues like tabbing goes to first heading of table and while using arrow keys, only the left arrow key works.can someone helps here
